modules/oe_content_extra_project/config/install/field.field.oe_organisation.oe_cx_project_stakeholder.oe_logo.yml

```diff
@@ -0,0 +1,34 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.storage.oe_organisation.oe_logo
+    - media.type.image
+    - oe_content_entity_organisation.oe_organisation_type.oe_cx_project_stakeholder
+  module:
+    - composite_reference
+third_party_settings:
+  composite_reference:
+    composite: false
+    composite_revisions: false
```

Comment:
Interesting: The original `oe_stakeholder` does not have the entries for `composite_reference`.

On https://www.drupal.org/project/composite_reference I read this:

> Composite reference fields ensure that referenced entities get deleted when the referencing entity is deleted.

Seems like a good functionality.
But I wonder, why is this not the case for original `oe_content_project`? 
In the original module, do we reuse stakeholder orgs?
If so, this is a noteworthy difference to the original module.

Reply:
It seems the "problem" is that composite_reference module adds noise in settings even for fields where it is disabled.
Not just here, but probably in any reference field on any website where the module is enabled.
Would be nice if it would prevent this.
https://www.drupal.org/project/composite_reference/issues/3271311
But ok, let's not block the PR for this.